spec/unit/postmark/account_api_client_spec.rb

```diff
@@ -448,6 +448,16 @@
       end
     end
 
+    describe '#verify_domain_custom_tracking' do
+      let(:response) { {"CustomTrackingVerified" => true } }
+
+      it 'performs a PUT request to /domains/:id/verify_domain_custom_tracking endpoint' do
```

Comment:
Is the description correct?

```suggestion
      it 'performs a PUT request to /domains/:id/verifycustomtracking endpoint' do
```

Reply:
Nice catch!